spark/agent/src/main/scala/za/co/absa/spline/harvester/builder/write/WriteNodeBuilder.scala

```diff
@@ -37,10 +37,11 @@ class WriteNodeBuilder(command: WriteCommand)
       append = command.mode == SaveMode.Append,
       id = id,
       childIds = childIds,
-      params = Map(
-        OperationParams.Name -> command.name,
-        OperationParams.DestinationType -> command.sourceIdentifier.format
-      ) ++ command.params
+      params = Map(command.params.toSeq: _*),
```

Comment:
Why create new map here?

Reply:
Because Spark returns a case insensitive map that created issues in the past. See #371